contracts/sophia.md

```diff
@@ -122,6 +122,82 @@ function root(t : tree('a)) : option('a) =
     Bin(_, v, _) => Some(v)
 ```
 
+### Maps and records
+
+A Sophia record type is given by a fixed set of fields with associated,
+possibly different, types. For instance
+```
+  type account = { name    : string,
+                   balance : int,
+                   history : list(transaction) }
+```
+
+Maps, on the other hand, can contain an arbitrary number of key-value bindings,
+but of a fixed type. The type of maps with keys of type `'k` and values of type
+`'v` is written `map('k, 'v)`. Key types are restricted to atomic types (`int`,
+`address`, `bool`, and `string`).
```

Comment:
uint?

Reply:
We only have `int` at the moment. I'll make sure to update it when we add `uint`s.